bip-schnorr.mediawiki

```diff
@@ -146,10 +146,10 @@ The algorithm ''Sign(sk, m)'' is defined as:
 * Let ''d = d' '' if ''has_square_y(P)'', otherwise let ''d = n - d' ''.
 * Let ''k' = int(hash<sub>BIPSchnorrDerive</sub>(bytes(d) || m)) mod n''<ref>Note that in general, taking the output of a hash function modulo the curve order will produce an unacceptably biased result. However, for the secp256k1 curve, the order is sufficiently close to ''2<sup>256</sup>'' that this bias is not observable (''1 - n / 2<sup>256</sup>'' is around ''1.27 * 2<sup>-128</sup>'').</ref>.
 * Fail if ''k' = 0''.
-* Let ''R = k'G''.
+* Let ''R = k'⋅G''.
 * Let ''k = k' '' if ''has_square_y(R)'', otherwise let ''k = n - k' ''.
 * Let ''e = int(hash<sub>BIPSchnorr</sub>(bytes(R) || bytes(P) || m)) mod n''.
-* Return the signature ''bytes(R) || bytes((k + ed) mod n)''.
+* Return the signature ''bytes(R) || bytes((k + e⋅d) mod n)''.
```

Comment:
So far we've only used `⋅` for EC multiplication. Should we use it for scalar multiplication as well?

Reply:
Oh I wasn't aware. If we'd add it for scalar multiplication then we would have to change many more different code locations in this PR. But there's also a few places where we don't put a dot for EC multiplication (f.e. bip-schnorr adaptor section, bip-taproot taproot tweak verification).

I think we should either use it for scalar multiplication as well or rename multi char variables (`sk`) to single char.